RENDER_DEPLOYMENT.md

```
@@ -0,0 +1,199 @@
+### Instance Type
+- **Free** (Select free tier - $0/month)
+
+## Step 5: Add Environment Variables
+
+Click **"Advanced"** and add these environment variables:
+
+```env
+NODE_ENV=production
+PORT=3001
+
+# Jira Configuration
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)